index.js

```diff
@@ -40,10 +35,11 @@ module.exports = {
 
     process.env.EMBER_CLI_INJECT_LIVE_RELOAD_PORT = options.liveReloadPort;
     process.env.EMBER_CLI_INJECT_LIVE_RELOAD_BASEURL = baseURL;
+    process.env.EMBER_CLI_INJECT_LIVE_RELOAD_JS_URL = options.liveReloadJsUrl || baseURL;
 
     app.use(baseURL + 'ember-cli-live-reload.js', function(request, response, next) {
       response.contentType('text/javascript');
-      response.send(self.dynamicScript(options));
+      response.send(self.dynamicScript());
```

Comment:
This point memory leakage.
https://auth0.com/blog/four-types-of-leaks-in-your-javascript-code-and-how-to-get-rid-of-them/